com.unity.netcode.gameobjects/Editor/HiddenScriptEditor.cs

```diff
@@ -21,14 +20,16 @@ public override void OnInspectorGUI()
         }
     }
 
+#if UNITY_UNET_PRESENT
     /// <summary>
     /// Internal use. Hides the script field for UNetTransport.
     /// </summary>
-    [CustomEditor(typeof(UNetTransport), true)]
+    [CustomEditor(typeof(Unity.Netcode.Transports.UNET), true)]
```

Comment:
This would need to be `Unity.Netcode.Transports.UNET.UNetTransport`... typeof() on a namespace is a compile error... but since `UNITY_UNET_PRESENT` is never defined in this assembly, that error is invisible because this is always excluded from compilation.